src/main.c

```diff
@@ -2582,6 +2638,14 @@ static lbm_value ext_set_fw_version(lbm_value *args, lbm_uint argn) {
     return VESC_IF->lbm_enc_sym_true;
 }
 
+// Called from Lisp to pass in the fault code of the bms.
+static lbm_value ext_bms_set_fault(lbm_value *args, lbm_uint argn) {
+    unused(argn);
```

Comment:
I'm pretty sure you should check that `argn > 0`, otherwise you'll be accessing random memory.

Reply:
I can try, but I thought I had some sort of an issue with it not passing for some reason.

Reply:
That'd be pretty bad though :sweat_smile: this needs to work...

Reply:
Yeah I should be able to check with a spare BMS even though I don't have a working board atm. I'll get back on this. The problem with it accessing uninitialized memory would only be if the function is called without a parameter...which it's only called one time with the parameter passed, so honestly I don't see a scenario where this would happen.